app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import shutil
```

Comment:
I think you don`t need that! Just use .write() and .read()

Reply:
I think write() and .read() is not as effective as using a ready-made method